src/cupbearer/scripts/train_detector.py

```diff
@@ -15,6 +15,10 @@ def main(cfg: Config):
         rng=jax.random.PRNGKey(cfg.seed),
     )
 
+    if not cfg.no_load and hasattr(cfg.task, "backdoor"):
```

Comment:
Similar to comment above: we might be able to move this into the `BackdoorDetection` task